crates/smdk/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "smdk"
+version = "0.0.0"
+edition = "2021"
+authors = ["Fluvio Contributors <team@fluvio.io>"]
+description = "SmartModule Development Kit"
+repository = "https://github.com/infinyon/fluvio"
+license = "Apache-2.0"
+publish = false
+
+
+
+[[bin]]
+name = "smdk"
+path = "src/main.rs"
+doc = false
+
+
+[dependencies]
+tracing = "0.1.19"
+clap = { version = "3.1.8", features = ["std", "derive"], default-features = false }
```

Comment:
it looks like we can upgrade this to 4.X.X

Reply:
Let's do in separate PR

Reply:
I created https://github.com/infinyon/fluvio/issues/2633 and marked as `good_first_issue` just in case anyone external want to contribute with that